packages/attribute-slicer-powerbi/powerbi-visuals-utils/type.ts

```diff
@@ -0,0 +1,4 @@
+// Wen't this route vs doing exports-loader in webpack config, to preserve typings
```

Comment:
comment typo.  Might also considering moving this comment to powerbi-visuals-utils/index.ts to better explain whats going on with the whole directory.

Reply:
👍